drake/examples/Acrobot/test/paramEstSyntheticData.m

```diff
@@ -1,26 +1,99 @@
 function paramEstSyntheticData
 
-oldpath = addpath(fullfile(pwd,'..'));
```

Comment:
i think you'll need this, although it's true we now use `addpathTemporary` instead.  See other test directory files.